docs-src/multiple-ns.md

```diff
@@ -0,0 +1,37 @@
+# Routes in multiple namespaces
+
+With the Service APIs, a single Gateway can target routes across multiple
+namespaces. 
+
+This guide assumes that you have installed Service APIs CRDs and a conformant
+controller.
+
+In the following example:
+
+- `acme-lb` GatewayClass: The GatewayClass responsible for satisfying Gateway
+  and Route resources.
+- `multi-ns-gateaway` Gateway: The Gateway is configured with a single listener
+  on port 80 which selects routes from **all namespaces** which have the label
+  `product: baz`. Notice how the `routes.namespaces.from` field in the listener
+  is set to `All`.
+- `service-apis-example-ns1` and `service-apis-example-ns2` Namespaces: These
+  are two namespaces in which we route resources are installed.
+- `http-app-1` and `http-app-2` HTTPRoutes: These are two resources that are
+  installed in separate namespaces. These routes will be picked up by
+  `multi-ns-gateway` resource for the following reasons:
```

Comment:
s/These routes will be picked up by `multi-ns-gateway` resource for the following reasons/These routes will be bound to Gateway `multi-ns-gateway` for the following reasons/